qemu/qmp.go

```diff
@@ -1439,6 +1440,12 @@ func (q *QMP) ExecuteNVDIMMDeviceAdd(ctx context.Context, id, mempath string, si
 			"share":    true,
 		},
 	}
+
+	if q.version.Major > 4 || (q.version.Major == 4 && q.version.Minor >= 1) {
+		props := args["props"].(map[string]interface{})
+		props["pmem"] = pmem
+	}
```

Comment:
Does adding this flag create any side effect when/if qemu is not builty with libpmem?

Reply:
If this flag is used and qemu wasn't built with this library (--enable-libpmem), qemu will fail

Reply:
this is documented here https://github.com/qemu/qemu/blob/master/docs/nvdimm.txt#L223